client/components/SearchResultList.jsx

```diff
@@ -1,17 +1,22 @@
 import React from 'react';
 import propTypes from 'prop-types';
 import SearchResult from './SearchResult.jsx';
+import '../styles/searchbar.css';
 
 const SearchResultList = props => (
-  <div>
+  <div className="search-bar-dropdown-container">
     {props.restaurants.map((restaurant, idx) => (
-      <SearchResult key={idx} restaurant={restaurant}/>
+      <SearchResult key={idx}
```

Comment:
Just a suggestion: The linter would suggest to use key="restaurant.id" over key="idx"